### PR TITLE
Save ordering between different use

### PR DESCRIPTION
**Pull request by darkkiss**
Save ordering between different use

### CODE REVIEW

**Comment by durunvo** [comment]
Is this in progress or done?

File: src/lib/ColumnPreferences.js [hunk 1]
```diff
@@ -43,6 +43,20 @@ export function getPreferences(appId, className) {
   }
 }
 
+export function getColumnSort(sortBy, appId, className) {
+  let objectName = className + ':' + 'columnSort';
```
Comment (durunvo) [inline src/lib/ColumnPreferences.js]: const objectName = `${className}:columnSort`
would be nice

Comment (darkkiss) [inline src/lib/ColumnPreferences.js]: Hello, I don't understand what do you mean by that ? Constant as a variable?

**Comment by peterdotjs** [comment]
@darkkiss awesome job for contributing! Have you tested this out? Currently for preferences we are storing the column width and order in an array. You'll need to add an additional `sort` field to the localStorage preference and ensure we are reading it.

If you take a quick look at the local storage you'll understand what I mean. Let me know if I can make anything more clear.

File: src/lib/ColumnPreferences.js [hunk 2]
```diff
@@ -43,6 +43,20 @@ export function getPreferences(appId, className) {
...
+  let cachedSort = getPreferences(appId, objectName) || [ { name: '-createdAt' } ];
+  let updated = false;
+  if(cachedSort != sortBy) {
```
Comment (peterdotjs) [inline src/lib/ColumnPreferences.js]: nit: !==

File: src/lib/ColumnPreferences.js (hunk 1 above)
Comment (peterdotjs) [inline src/lib/ColumnPreferences.js]: I think `columnSort` as a `const` placed at the top would be nice. Consistent with our existing style: https://github.com/ParsePlatform/parse-dashboard/blob/master/src/lib/ColumnPreferences.js#L79

**Comment by peterdotjs** [comment]
Just a couple nits but otherwise looks and works great.

**Comment by mmarshak** [comment]
I am using Chrome and I have dashboard version 1.0.22, and the order of the columns is not being saved between different launches of the browser.
Do I need to do something to ask dashboard to save the order?

**Pull request merged**